sim/src/sim.rs

```diff
@@ -531,13 +531,7 @@ impl Sim {
                 );
             }
             Command::UpdateLaggyHead(car) => {
```

Comment:
I see this term a few places in this PR: What's a "laggy head"?

Reply:
https://dabreegster.github.io/abstreet/trafficsim/discrete_event.html#exact-positions
https://docs.google.com/presentation/d/1nUodhr42eppB2E2eMAnuTkMhIVuHnN7_6i6V6MA028c/edit#slide=id.g830f949219_1_15
It's so hard to get motivated to organize docs >_<  Really need to consolidate / update the mdbook

Reply:
Thanks for the links!